### PR TITLE
Code along

### PR DESCRIPTION
**Pull request by raynamharris**
Okay, here is my attempt to replace _most_ instances of "live coding" with "participatory live coding". I say "most" because there any many instances where live coding is indeed the best option because they are describing a situation where, indeed, no one is "coding along" with the live coder in real time. 

I updated all files in `_episodes` and `_extras` and I added a definition to the glossary. 

this relates to https://github.com/carpentries/instructor-training/issues/776

### CODE REVIEW

File: _episodes/08-motivation.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ for learner success. As instructors, it is crucial to establish the workshop set
 as a safe space for learning. Establishing a safe space for learning is a combination 
 of many factors:
 
-+ *Presenting the instructor as a learner*. Admitting that you don't know everything is part of showing that it is acceptable to make mistakes, and encouraging a growth mindset in learners (we'll talk much more about growth mindset in [a later lesson]({{ page.root }}/09-mindset/)). Using live coding in teaching concepts is very useful for this reason. It is common to make errors while live coding. When done well, these errors can be very instructive for novice learners.
++ *Presenting the instructor as a learner*. Admitting that you don't know everything is part of showing that it is acceptable to make mistakes, and encouraging a growth mindset in learners (we'll talk much more about growth mindset in [a later lesson]({{ page.root }}/09-mindset/)). Using participatory live coding in teaching concepts is very useful for this reason. It is common to make errors while participatory coding. When done well, these errors can be very instructive for novice learners.
```
Comment (karenword) [inline _episodes/08-motivation.md]: I wonder if the second "participatory" is necessary here? It's a bit of a mouthful, so I think it's best to keep it only where it adds a meaningful distinction.

File: _episodes/08-motivation.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ for learner success. As instructors, it is crucial to establish the workshop set
 as a safe space for learning. Establishing a safe space for learning is a combination 
 of many factors:
 
-+ *Presenting the instructor as a learner*. Admitting that you don't know everything is part of showing that it is acceptable to make mistakes, and encouraging a growth mindset in learners (we'll talk much more about growth mindset in [a later lesson]({{ page.root }}/09-mindset/)). Using live coding in teaching concepts is very useful for this reason. It is common to make errors while live coding. When done well, these errors can be very instructive for novice learners.
++ *Presenting the instructor as a learner*. Admitting that you don't know everything is part of showing that it is acceptable to make mistakes, and encouraging a growth mindset in learners (we'll talk much more about growth mindset in [a later lesson]({{ page.root }}/09-mindset/)). Using participatory live coding in teaching concepts is very useful for this reason. It is common to make errors while coding. When done well, these errors can be very instructive for novice learners.
```
Comment (lexnederbragt) [inline _episodes/08-motivation.md]: Here it is also mentioned before the lesson introducing it, so it need a tiny bit of explaining.

```suggestion
+ *Presenting the instructor as a learner*. Admitting that you don't know everything is part of showing that it is acceptable to make mistakes, and encouraging a growth mindset in learners (we'll talk much more about growth mindset in [a later lesson]({{ page.root }}/09-mindset/)). Using [participatory live coding]({{ page.root }}/14-live/), our chosen method for teaching concepts, is very useful for this reason. It is common to make errors while coding. When done well, these errors can be very instructive for novice learners.
```

File: _episodes/11-practice-teaching.md [hunk 3]
```diff
@@ -107,8 +107,9 @@ Now that you've had some practice observing teaching and giving feedback, let's
 > 3.  Get together with your group and have each person teach their segment to the group,
 >     while one person records this (video and audio)
 >     using a cell phone or some other handheld device.
->     Use a whiteboard or other visual aids if available, but do not use live coding. 
->     We'll be practicing with live coding later.
+>     Use a whiteboard or other visual aids if available
+>     (Note: Do not live code you lesson.
+>     There will be a change to live code later.)
```
Comment (lexnederbragt) [inline _episodes/11-practice-teaching.md]:
```suggestion
>     There will be a chance to live code later.)
```

File: _episodes/14-live.md [hunk 4]
```diff
@@ -1,38 +1,43 @@
...
 Instead, instructors plug their laptop into the projector
 and work through the lesson,
 typing in the code,
 reformatting data,
 and talking as we go.
+This is called ["live coding"](https://en.wikipedia.org/wiki/Live_coding).
+However, the instructor is not live coding in a vacuum. 
+Importantly, students are strongly encouraged
+to "code-along" with the instructor. 
+We refer to the practice of having the instructor live code and the learners follow along as "participatory live coding".
```
Comment (lexnederbragt) [inline _episodes/14-live.md]:
```suggestion
We refer to the practice of having the instructor live code and the learners code along as "participatory live coding".
```
Could consider adding " or, less formally, 'code-along sessions'. "

Comment (raynamharris) [reply _episodes/14-live.md]: Thanks! I incorporated both suggestions.

**Comment by raynamharris** [comment]
Thanks all! I've accepted all Lex's changes. We should be good to go now :)

**Comment by lexnederbragt** [comment]
Great! Can I have the pleasure of merging this one? :-)

**Pull request merged**